jmh.gradle

```diff
@@ -26,11 +26,11 @@ def scalaVersion = System.getProperty("scalaVersion") != null ? System.getProper
 def jmhProjects = []
 
 if (jdkVersion == '8' && sparkVersions.contains("2.4")) {
-  jmhProjects.add(project(":iceberg-spark:iceberg-spark2"))
+  jmhProjects.add(project(":iceberg-spark:iceberg-spark-2.4"))
```

Comment:
Should we also add the scala version here?

Reply:
As we've discussed in this conversation https://github.com/apache/iceberg/pull/4158#issuecomment-1045995498,  spark 2.4 should be scala-free, which means it won't add any scala code or pull in any transitive scala dependency for downstream iceberg-spark-2.4 users.   So in theory we should not add the scala suffix version to the iceberg-spark-2.4 artifacts because the same artifact should be work for both spark with scala 2.12 and spark with scala 2.11.